support/investigating-pod-issues.adoc

```diff
@@ -0,0 +1,30 @@
+[id="investigating-pod-issues"]
+= Investigating pod issues
+include::modules/common-attributes.adoc[]
+:context: investigating-pod-issues
+
+toc::[]
+
+{product-title} {product-version} leverages the Kubernetes concept of a pod, which is one or more containers deployed together on one host. A pod is the smallest compute unit that can be defined, deployed and managed on {product-title} {product-version}.
```

Comment:
Missing serial comma. Also, you don't need {product-version} at the conceptual level such as here, IMO.

Reply:
Resolved.